lang/en/history.php

```diff
@@ -20,6 +20,10 @@
         'delete'    => ':user deleted :entity',
         'restore'   => ':user restored :entity',
         'update'    => ':user updated :entity',
+        'create_post'    => ':user created post ":post" in :entity',
```

Comment:
`:user create the post ":post" on :entity` (missing `the` and replace `in` with `on` on all 3 keys